blackbox/docs/admin/system-information.rst

```diff
@@ -1015,38 +1015,46 @@ Jobs, Operations, and Logs
-operations adds a slight performance overhead.
+Jobs and operations that finished executing are additionally recorded in
+memory. There are two retention policies available to control how many records
+should be kept.
 
-The tracking jobs are stored in memory. The amount of memory that can be
-used to store the jobs is limited by the
-:ref:`Jobs Log Circuit Breaker <stats.breaker.log.jobs.limit>` and
-:ref:`Operations Log Circuit Breaker <stats.breaker.log.operations.limit>`.
-If the memory limit is reached, an error message will be logged and the log
-table will be cleared completely.
-
-You can avoid the tables being cleared completely by configuring a records
-retention policy. We currently support two retention policies.
-
-One option is to configure how many records are to be stored in the logs
-tables. This is configurable using :ref:`Jobs Log Size <stats.jobs_log_size>`
-and :ref:`Operations Log Size <stats.operations_log_size>`. Once the configured
-table size is reached the older log records are deleted as newer records are
-added.
+One option is to configure the maximum number of records which should be kept.
+Once the configured table size is reached the older log records are deleted as
+newer records are added. This is configurable using :ref:`Jobs Log Size
+<stats.jobs_log_size>` and :ref:`Operations Log Size
+<stats.operations_log_size>`. 
 
 Another option is to configure an expiration time for the records. In this
 case, the records in the logs tables are periodically cleared if they are older
 than the expiry time. This behaviour is configurable using
 :ref:`Jobs Log Expiration <stats.jobs_log_expiration>` and
 :ref:`Operations Log Expiration <stats.operations_log_expiration>`.
 
-If you want to use the jobs and operations tables, you must enable the
-collection of CrateDB statistics with :ref:`ref-set`, like so::
 
-    cr> set global stats.enabled = true;
-    SET OK, 1 row affected (... sec)
+In addition to these retention policies there is a memory limit in place
+preventing these tables from taking up too much memory.
+The amount of memory that can be used to store the jobs can be configured
+using :ref:`Jobs Log Circuit Breaker <stats.breaker.log.jobs.limit>` and
+:ref:`Operations Log Circuit Breaker <stats.breaker.log.operations.limit>`.
+If the memory limit is reached, an error message will be logged and the log
+table will be cleared completely.
+
+
+It is also possible to define a filter which must match in order for jobs to be
+recorded after they finished executing.
+This can be useful to only record slow queries or queries that failed due to an
+error.
+This filter can be configured using the :ref:`Jobs Log Filter
+<stats.jobs_log_filter>` setting.
+
+Furthermore, there is a second filter setting which also results in a log entry
+for all finished jobs that match this filter. This can be configured using
+:ref:`Persistent Jobs Log Filter <stats.jobs_log_persistent_filter>`. 
```

Comment:
in general I think it would be good to reference the settings by name, as literals, as they appear in the file, instead of giving them capitalized titles

this applies to all such titling in the changeset